twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -462,6 +524,18 @@ scrooge_scala_import = rule(
                 Label(
                     "//external:io_bazel_rules_scala/dependency/thrift/scrooge_core",
                 ),
+                Label(
```

Comment:
I actually don't think these are used, are they? On the import target there is no compilation.

Reply:
Will remove these. I'm guessing that if we're importing external libraries they should have the `runtime_deps` set properly